### PR TITLE
Implement draw string callers

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: src/OpenLoco/Graphics/Gfx.cpp [hunk 1]
```diff
@@ -971,17 +972,68 @@ namespace OpenLoco::Gfx
...
+        else if (_currentFontSpriteBase == Font::small)
+            lineHeight = 6;
+        else if (_currentFontSpriteBase == Font::large)
+            lineHeight = 18;
+
+        _currentFontFlags = 0;
+        int16_t leftoverHeight = breakCount * (lineHeight / 2);
+        Ui::Point point = { x, y };
+        char* ptr = buffer;
+        while (true)
```
Comment (duncanspumpkin) [inline src/OpenLoco/Graphics/Gfx.cpp:995]: Can you tidy this function up. It's looping through each line printing each one as it goes.

File: src/OpenLoco/Graphics/Gfx.cpp [hunk 2]
```diff
@@ -971,17 +1005,34 @@ namespace OpenLoco::Gfx
...
+
+        _currentFontFlags = 0;
+        Ui::Point point = { x, y };
+        const char* ptr = buffer;
+
+        for (auto i = 0; ptr != nullptr && i <= breakCount; i++)
+        {
+            Gfx::drawString(context, point.x, point.y, colour, const_cast<char*>(ptr));
+            ptr = advanceToNextLine(ptr);
+            point.y += lineHeight;
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Graphics/Gfx.cpp:1032]: This is losing the colour used to paint. For example, the first line might switch to painting using a window colour, which the successive lines then do not use.

Comment (duncanspumpkin) [reply src/OpenLoco/Graphics/Gfx.cpp:1032]: It is losing colour as the original used to run the drawString function once to set the global colours then for the looping it passed in 0xFE for colour which is a special code to tell it not to set the global colours at the start.

Comment (duncanspumpkin) [reply src/OpenLoco/Graphics/Gfx.cpp:1032]: Can be fixed by adding the following on L1012:
```
Gfx::drawString(context, context.x, context.y, colour, "");
```
and modifying L1030 to be:
```
Gfx::drawString(context, point.x, point.y, AdvancedColour::FE(), const_cast<char*>(ptr));
```

File: src/OpenLoco/Windows/News/News.h [hunk 3]
```diff
@@ -8,7 +8,6 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Ui::Windows::NewsWindow
 {
-    static loco_global<uint8_t[3], 0x005215B5> _unk_5215B5;
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/News/News.h:11]: for reference 5215B5 is an empty string used for priming the text colour global var.

**Review by AaronVanGeffen — COMMENTED** [review]
Thanks for the patches, @duncanspumpkin. I'm fine with this PR in its current state. Do you think this is ready to be merged, too?

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**